libs/cases/frontend/view/src/lib/components/cases-brows/case-brows.component.scss

```diff
@@ -0,0 +1,35 @@
+.table {
+	color: #777784;
+	& .emergency {
+		background-color: #ff26401a;
+		color: #ffffffbf;
+	}
+
+	& .status-open {
+		color: #ffffffbf;
+	}
+
+	& .status-inprogress {
+		color: #ff9000;
+	}
+
+	& .status-overdue {
+		color: #ff2640;
+	}
+
+	& .status-onhold {
+		color: #00b5d1;
+	}
+
+	& .status-done {
+		color: #01c151;
+	}
+
+	& .status-cancelled {
+		color: #777784;
+	}
+
+	& .status-archived {
+		color: #777784;
+	}
+}
```

Comment:
Why dont you add this in a shared lib? To me it seems to be important for other classes as well. Additionally .status-archieved and cancled dont have an impact on the styling They are already set.

Reply:
I agree that would be better and I can take a look at it after IP meeting.